pkg/api/serialization_test.go

```diff
@@ -298,11 +298,13 @@ func fuzzInternalObject(t *testing.T, forVersion schema.GroupVersion, item runti
 		},
 		func(j *build.DockerBuildStrategy, c fuzz.Continue) {
 			c.FuzzNoCustom(j)
-			j.From.Kind = "ImageStreamTag"
-			j.From.Name = "image:tag"
-			j.From.APIVersion = ""
-			j.From.ResourceVersion = ""
-			j.From.FieldPath = ""
+			if j.From != nil {
```

Comment:
good question. But, I guess it should be possible to be nil, like the other ObjectReferences. It's a pointe for ages: 8f01437071bc77e7878dae263411a4dd2303915c. @bparees any idea?